nixos/modules/services/networking/tinyproxy.nix

```
@@ -4,9 +4,11 @@ with lib;
 
 let
   cfg = config.services.tinyproxy;
+  containsPath = value: if builtins.isString value then lib.hasPrefix "/" value else false;
   mkValueStringTinyproxy = with lib; v:
         if true  ==         v then "yes"
         else if false ==    v then "no"
+        else if containsPath v then ''"${v}"''
```

```suggestion
        else if types.path.check v then ''"${v}"''
```

Where did you find this `containsPath` anyway?

---

The containsPath is the function I've defined above :)
It's far from perfect but tinyproxy has a weird config file syntax. 
I'll test your proposal but I'm not sure it will work because I tried with some type checking and since it's not really a path but a string containing a path it was not working.

---

It works like what you did:

https://github.com/NixOS/nixpkgs/blob/7727237a78b92a32a8cea8396ea0a011217e13ee/lib/types.nix#L517-L522

---

Indeed ! Just always hard to find the existing function in nix. 
I'll change that.